README.md

```diff
@@ -145,74 +152,27 @@ sumAgg := view.SumAggregation{}
 meanAgg := view.MeanAggregation{}
 ```
 
-### Creating, registering and unregistering a view
-
-Create and register a view:
+Here we create a view with the DistributionAggregation over our Measure.
+Setting tags to nil (as in this example) means that Measurements with all tags
```

Comment:
`that all Measurements will be aggregated together, irrespective of their tags (i.e. no grouping by tag)`